polkadb/database.go

```diff
@@ -14,7 +14,7 @@ type BadgerDB struct {
 	db   *badger.DB
 }
 
-// Iterator struct contains a transaction, iterator and context fields released, initialized
+// Iterate struct contains a transaction, iterator and context fields released, initialized
 type Iterate struct {
```

Comment:
I think this should be `Iterable`, `Iterate` is not a noun.

Reply:
changed